.travis.yml

```diff
@@ -4,8 +4,6 @@ dist: xenial
 language: python
 matrix:
   include:
-    - python: 2.6
```

Comment:
Why did you remove this? It is an unrelated change for the new bond policy.

Reply:
Hi @tyll , sorry that was me just mucking around as the py2.6 test was failing with PyYAML and deprecated version. I've fixed this with my latest change to this pull request.